docker-generator/src/main/java/org/ballerinax/docker/generator/DockerArtifactHandler.java

```diff
@@ -158,6 +158,10 @@ private DockerClient createClient() {
             dockerClientConfig.withRegistryPassword(this.dockerModel.getPassword());
         }
 
+        if (null != this.dockerModel.getDockerConfig()) {
```

Comment:
Should we move this to line before  #131 ? . Then whatever that is given from the annotations will override.

Reply:
Values given in the annotations should have the priority.